core-war/src/main/java/org/silverpeas/web/jobdomain/control/JobDomainPeasSessionController.java

```diff
@@ -787,7 +790,14 @@ public void importCsvUsers(FileItem filePart, UserRequestData data, HttpServletR
       userRequestData.setSendEmail(data.isSendEmail());
       userRequestData.setExtraMessage(data.getExtraMessage());
       userRequestData.setUserManualNotifReceiverLimitEnabled(true);
-      createUser(userRequestData, properties, req);
+      try {
+        createUser(userRequestData, properties, req);
+      } catch (JobDomainPeasException e) {
+        JobDomainPeasTrappedException ee = new JobDomainPeasTrappedException(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Immediately throw this expression instead of assigning it to the temporary variable "ee". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1488)